Assignment4/miguelventoso - 21A284/Task06.py

```diff
@@ -0,0 +1,92 @@
+# %%
+"""
+**Task 06: Modifying RDF(s)**
+"""
+
+# %%
+!pip install rdflib 
```

Comment:
This will not run in Python, you need to comment out this line. The same applies for other files